tasks/build-css.js

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env node
+
+/*
+Copyright 2022 Adobe. All rights reserved.
```

Comment:
I'd like to see these scripts migrated into the tools/component-builder and tools/component-builder-simple workspaces along with their dependencies. It keeps assets scoped to what they are being used for and make dependency management easier.